utils/appliance/__init__.py

```diff
@@ -105,14 +97,14 @@ def __init__(self, address=None, browser_steal=False, container=None):
         # ** Note also the complexity surrounding this that browser_session is a "new" session
         # ** and that we still need to work out how to handle multiple browsers, for multiple
         # ** appliances and how that interacts with Wharf etc.
-        self.browser = UIEndpoint('ui', 'browser_session', self)
-        self.db_session = DBEndpoint('db', 'db', self)
+        self.browser = ViaUI(owner=self)
+        self.db_session = ViaDB(owner=self)
         # ** this will be renamed <---- see I told you we had it covered.
         self.sentaku_ctx = sentaku.ImplementationContext(
-            implementations=sentaku.AttributeBasedImplementations(self, {
-                ViaDB: 'db_session',
-                ViaUI: 'browser',
-            }),
+            implementations={
```

Comment:
before we used lazy attributes directly, now that we use implementation objects that we do not have to obtain lazy, we can use them directly